test/utilities/dkg-client-helper.mjs

```diff
@@ -9,17 +9,19 @@ class DkgClientHelper {
         return this.client.node.info();
     }
 
-    async publish(data, wallet, hubContract) {
+    async publish(data, wallet) {
         const options = {
             visibility: 'public',
-            holdingTimeInYears: 1,
+            triplesNumber: 3,
```

Comment:
we should modify the client to calculate triples and chunks. It shouldnt be passed as an optional parameter